lib/jdomodels/src/main/java/org/sagebionetworks/evaluation/dao/EvaluationDAOImpl.java

```diff
@@ -177,7 +175,7 @@ public Evaluation get(String id) throws DatastoreException, NotFoundException {
 
 
 	@Override
-	public List<Evaluation> getAccessibleEvaluationsForProject(String projectId, List<Long> principalIds, ACCESS_TYPE accessType, long limit, long offset) 
+	public List<Evaluation> getAccessibleEvaluationsForProject(String projectId, List<Long> principalIds, ACCESS_TYPE accessType, Long now, long limit, long offset) 
```

Comment:
Please rename "now" to a param name that suggests that it's applying a time filter.